src/Control/Inciter/Types.hpp

```diff
@@ -161,17 +156,19 @@ using TransportPDEParameters = tk::TaggedTuple< brigand::list<
 
 //! Compressible flow equation parameters storage
 using CompFlowPDEParameters = tk::TaggedTuple< brigand::list<
-    tag::depvar,                std::vector< char >
-  , tag::physics,               std::vector< PhysicsType >
-  , tag::problem,               std::vector< ProblemType >
-  , tag::bcdir,                 std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
-  , tag::bcsym,                 std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
-  , tag::bcinlet,               std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
-  , tag::bcsubsonicoutlet,      std::vector< std::vector<
-                                kw::sideset::info::expect::type > >
+    tag::depvar,        std::vector< char >
+  , tag::physics,       std::vector< PhysicsType >
+  , tag::problem,       std::vector< ProblemType >
+  , tag::bcdir,         std::vector< std::vector<
+                          kw::sideset::info::expect::type > >
+  , tag::bcsym,         std::vector< std::vector<
+                          kw::sideset::info::expect::type > >
+  , tag::bcinlet,       std::vector< std::vector<
+                          kw::sideset::info::expect::type > >
+  , tag::bcoutlet,      std::vector< std::vector<
```

Comment:
Also renamed this. @adityakpandare is that okay with you? I thought this was okay because there is not other "outlet" bc here.

Reply:
I think we should keep it as ```bcsubsonicoutlet```. We will soon have a characteristic outlet BC which detects if the flow at the boundary-face is sub/super sonic and behaves accordingly. Does that make sense?

Reply:
Yeah okay. Or we can rename this when we implement that. I'm fine either way.